### ISSUE TITLE
CapnP' Cerealization and IPC

### ISSUE THREAD

**Issue by itdaniher** [ade-ma/LibRedio#14]
The current plan is to have a final-level sink / tail that handles IPC via Unix domain socket in its own task, sending CapnP serialized messages. Python will then append these messages to a log file, push them to a browser via websockets(?), and do math(?).

The schema's been written, capnp-rust docs have been read, and rust IPC socket docs have been read, but Ian needs to actually sit down and write some code. This is slightly blocked by https://github.com/ade-ma/LibRedio/issues/5.

/cc @madisonmay
